profiles/arch/alpha/package.use.mask

```diff
@@ -1,6 +1,10 @@
 # Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
+# Michał Górny <mgorny@gentoo.org> (07 Aug 2017)
+# Requires USE=networkmanager that is masked in this profile.
+app-misc/geoclue skyhook
```

Comment:
I know but I don't think it's really useful to put slot in this file, and the issue remains while the slot is kept.